Floofbot/Modules/Utilities.cs

```diff
@@ -139,5 +141,60 @@ public async Task ServerInfo()
             await Context.Channel.SendMessageAsync("", false, embed.Build());
 
         }
+        [Command("serverlist")]
+        [Summary("Returns a list of servers that the bot is in")]
+        [RequireOwner]
+        public async Task ServerList()
+        {
+            List<SocketGuild> guilds = new List<SocketGuild>(Context.Client.Guilds);
+            List<PaginatedMessage.Page> pages = new List<PaginatedMessage.Page>();
+
+            foreach (SocketGuild g in guilds)
+            {
+                List<EmbedFieldBuilder> fields = new List<EmbedFieldBuilder>();
+
+                fields.Add(new EmbedFieldBuilder()
+                {
+                    Name = $"Owner",
+                    Value = $"{g.Owner.Username}#{g.Owner.Discriminator} | ``{g.Owner.Id}``",
```

Comment:
Sanity check: is this the server owner or the bot owner?
Maybe change the name field to "Server Owner"

Reply:
Server owner